docs/getting-started/csm-stack/knoten-camera-tutorial.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Generating a CSM Camera Model from PDS Image in Knoten with Basic Camera Operations\n",
+    "\n",
+    "## 1. Prerequisites, Install Knoten\n",
+    "This tutorial requires Knoten version 0.4 or higher.\n",
+    "\n",
+    "`conda install -c conda-forge knoten=0.4`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Create a CSM camera for a PDS image"
```

Comment:
I ran into a `matplotlib` import issue in this cell and had to install `matplotlib` separately after running the `conda install` command above in a new conda environment. Do we want to add another install step above for `matplotlib`?

Reply:
Just took a look and matplotlib is listed in knoten's environment.yml so it should have been included in the knoten build. Not sure what happened. I'll look into this.